account_payment_order/tests/test_payment_order_inbound.py

```diff
@@ -136,3 +137,25 @@ def test_creation(self):
         payment_order.cancel2draft()
         payment_order.unlink()
         self.assertEqual(len(self.payment_order_obj.search(self.domain)), 0)
+
+    def test_payment_partner_bank_value(self):
+        payment_order = self.inbound_order
+        self.assertEqual(len(payment_order.ids), 1)
+
+        payment_order.write({"journal_id": self.journal.id})
+
+        self.assertEqual(len(payment_order.payment_line_ids), 1)
+        self.assertFalse(payment_order.payment_ids)
+
+        # Open payment order
+        payment_order.draft2open()
+
+        self.assertEqual(len(payment_order.payment_ids), 1)
+        payment = payment_order.payment_ids[0]
+        self.assertEqual(len(payment.payment_line_ids), 1)
+        payment_line = payment.payment_line_ids[0]
+        self.assertEqual(payment.partner_bank_id, payment_line.partner_bank_id)
+
+        # Assert payment line's bank has not changed
+        payment._compute_available_partner_bank_ids()
```

Comment:
Tests shouldn't launch manually private or compute methods. They should be triggered through business flows.

Reply:
Modified. I'm not calling the `compute` method directly.